course_project/20/main/java/com/example/Twitter/TweetsController.java

```diff
@@ -0,0 +1,48 @@
+package com.example.Twitter;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Map;
+
+@RestController
+public class TweetsController {
+
+    @Autowired
+    TweetRepository tweetRepository;

```

Comment:
Not a good idea to have repository in controller directly, what if you want to do something else than storing the tweet for example (send email after tweet is stored)?